src/main/java/org/gridsuite/ds/server/service/DynamicSimulationResultService.java

```diff
@@ -136,4 +145,12 @@ public DynamicSimulationStatus findStatus(UUID resultUuid) {
                 .map(ResultEntity.BasicFields::getStatus)
                 .orElse(null);
     }
+
+    @Override
+    public String findDebugFileLocation(UUID resultUuid) {
```

Comment:
```suggestion
    public String getDebugFileLocation(UUID resultUuid) {
```

Reply:
The method returns null instead of throwing an exception when the corresponding entity isn't found.

Since the null check is already managed in the abstract worker class, I believe findDebugFileLocation is a more suitable name than getDebugFileLocation in this context.